relay-system/src/controller.rs

```diff
@@ -152,8 +183,8 @@ impl fmt::Debug for Controller {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("Controller")
             .field("timeout", &self.timeout)
-            .field("subscribers", &self.subscribers.len())
-            .finish()
+            .field("subscribers", &self.subscribers.len()) //  Ask if we need to update these here
```

Comment:
The new mechanism doesn't really allow for showing anything useful about the subscribers/receivers so might as well leave this out and leave this as is.